WS2012R2/lisa/xml/RedHat_test_kernel.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ Linux on Hyper-V and Azure Test Code, ver. 1.0.0
+ Copyright (c) Microsoft Corporation
+
+ All rights reserved.
+ Licensed under the Apache License, Version 2.0 (the ""License"");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+ OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION
+ ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR
+ PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+
+ See the Apache Version 2.0 License for specific language governing
+ permissions and limitations under the License.
+-->
+
+<config>
+    <global>
+        <logfileRootDir>TestResults</logfileRootDir>
+        <defaultSnapshot>ICABase</defaultSnapshot>
+        <email>
+            <recipients>
+                <to>myself@mycompany.com</to>
+            </recipients>
+            <sender>myself@mycompany.com</sender>
+            <subject>Redhat test kernel update Test Results</subject>
+            <smtpServer>mysmtphost.mycompany.com</smtpServer>
+        </email>
+       <!-- Optional testParams go here -->
+    </global>
+
+    <testSuites>
+        <suite>
+            <suiteName>Redhat</suiteName>
+            <suiteTests>
+                <suiteTest>redhat_update_kernel</suiteTest>
+            </suiteTests>
+        </suite>
+    </testSuites>
+
+    <testCases>
+        <test>
+            <testName>redhat_update_kernel</testName>
+            <testScript>setupscripts\kernel.ps1</testScript>
+            <files>remote-scripts\ica\redhat_upgrade_kernel.sh,remote-scripts\ica\redhat_downgrade_kernel.sh,remote-scripts\ica\utils.sh</files>
+            <testParams>
+                <!--<param>URL=http://PATH/TO/TEST/KERNEL</param>-->
+                <param>URL=http://10.66.10.35/pub/kernel/</param>
+                <param>DOWNKERNEL=kernel-3.10.0-305.el7.x86_64.rpm</param>
+                <param>UPKERNEL=kernel-abi-whitelists-3.10.0-364.el7.noarch.rpm,kernel-3.10.0-364.el7.x86_64.rpm</param>
```

Comment:
my concern here is that the script is specifically targeting 7.2 only.
if this can be made more generic to run at least on any 7.x or even some of the latest 6.x (6.7 maybe), it would make the test more broadly to be used.